fala/templates/footer.html

```diff
@@ -46,7 +46,7 @@ <h2 class="govuk-visually-hidden">Support links</h2>
         <a class="govuk-footer__link" href="https://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/" rel="license">Open Government Licence v3.0</a>, except where otherwise stated
       </span>
     </div>
-    <div class="govuk-footer__meta-item">
```

Comment:
I think we should be keeping this?

Reply:
@starswan yes you're right. I think I changed the CSS to see the difference and then forgot to put back.